src/quo2/components/tags/context_tag/view.cljs

```diff
@@ -0,0 +1,85 @@
+(ns quo2.components.tags.context-tag.view
```

Comment:
Moved the previos code from `quo2.components.tags.context-tags` -> `quo2.components.tags.context-tag.view`